modules/dxp/apps/commerce/commerce-machine-learning-impl/src/main/java/com/liferay/commerce/machine/learning/internal/forecast/search/index/CommerceMLForecastIndexer.java

```diff
@@ -6,44 +6,36 @@
 package com.liferay.commerce.machine.learning.internal.forecast.search.index;
 
 import com.liferay.commerce.machine.learning.internal.search.api.CommerceMLIndexer;
-import com.liferay.commerce.machine.learning.internal.search.index.helper.CommerceMLSearchEngineHelper;
-import com.liferay.portal.search.index.IndexNameBuilder;
+import com.liferay.commerce.machine.learning.internal.search.index.helper.BaseCommerceMLIndexer;
 
 import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Riccardo Ferrari
  */
 @Component(service = CommerceMLIndexer.class)
-public class CommerceMLForecastIndexer implements CommerceMLIndexer {
+public class CommerceMLForecastIndexer
+	extends BaseCommerceMLIndexer implements CommerceMLIndexer {
```

Comment:
Hey @MrJohn1911, please check out all the `CommerceMLIndexer` implementations and rename them. They are not following the naming pattern.